src/configurations/destinations/rs/db-config.json

```diff
@@ -68,6 +68,7 @@
         "jsonPaths",
         "useRudderStorage",
         "underscoreDivideNumbers",
+        "cleanupObjectStorageFiles",
```

Comment:
Currently I've made changes only for Redshift. If the changes are good, then I'll update the PR for other warehouse destinations as well

Reply:
Added for other warehouses as well